scripts/linter/index.js

```diff
@@ -34,10 +34,16 @@ function main() {
 
     const reportedFiles = [];
 
-    for (const fp of walkDocs()) {
-        const file = vfile.readSync(fp);
-        markdownParser().processSync(file);
-        reportedFiles.push(file);
+    if (args.length === 0) {
+        args.push(undefined); // use default starting directory
```

Comment:
I would much rather you edit `walk-docs.js` to not have a default location and move that here to the `main()` function. Then `walk-docs.js` becomes more agnostic we won't need this "hack" here.